SpinCore_pp/pulse_length_conv.py

```diff
@@ -5,69 +5,97 @@
 
 def prog_plen(desired_actual):
     """
-    Takes the desired pulse length and tells the
-    user what pulse length should be programmed in order to get the actual desired
-    pulse length
+    Takes the desired beta (us sqrt(W)) and tells the
+    user what beta should be programmed in order to get the actual desired
+    beta
     Parameters
     ==========
     desired_actual: float
-                    the actual pulse length you wish the spincore to output,
-                    in us
+                    the actual beta you wish the spincore to output,
+                    in us*sqrt(W)
     Returns
     =======
     retval: float
-            The pulse length you tell spincore in order to get the desired actual.
+            The beta you tell spincore in order to get the desired actual.
     """
-    # {{{ list of programmed p90, actual p90 and actual 180 - used in
+    # {{{ list of programmed p90, actual p90*sqrt(W) and actual p180*sqrt(W) - used in
     # generating the calibrated fit
-    # list of the programmed pulse length, the actual p90 length
-    # and the actual p180 length based on the programmed p90
+    # list of the programmed pulse length, the actual p90*sqrt(P)
+    # and the actual p180*sqrt(P) based on the programmed p90
     datapoints = [
-        (1, 2.25869, 5.75412),
-        (2, 5.78065, 16.6168),
-        (3, 10.4132, 36.0229),
-        (3.5, 13.2053, 49.4104),
-        (4, 16.9808, 63.7255),
-        (4.5, 20.4275, 77.5474),
-        (5, 24.8538, 91.5177),
-        (5.5, 30.0159, 106.302),
-        (6, 35.6903, 119.849),
-        (6.5, 42.2129, 134.328),
-        (7, 48.9577, 149.029),
-        (7.5, 55.977, 162.774),
-        (8, 63.0445, 176.917),
-        (8.5, 69.881, 190.167),
-        (9, 76.9727, 203.486),
-        (9.5, 84.5149, 217.965),
-        (10, 91.7002, 232.042),
-        (10.5, 98.6599, 245.213),
-        (11, 105.327, 258.004),
-        (11.5, 112.985, 272.776),
-        (12, 120.264, 286.884),
-        (12.5, 126.251, 299.009),
+        (10, 1.6816, 4.3987),
+        (11.7996, 2.0908, 5.6696),
+        (13.60006, 2.5583, 7.0707),
+        (15.3997, 3.0091, 8.6477),
+        (17.2001, 3.5688, 10.3898),
+        (18.9997, 4.0651, 12.4032),
+        (20.8002, 4.5997, 14.5975),
+        (22.5998, 5.2121, 17.1804),
+        (24.40027, 5.8534, 20.1967),
+        (26.19987, 6.5583, 23.2631),
+        (28.0003, 7.2702, 26.7712),
+        (29.7999, 8.0251, 30.3686),
+        (31.6004, 8.9074, 34.5012),
+        (33.4, 9.7384, 38.7034),
```

Comment:
new values acquired today where the first value is beta = `linspace(10,100,50)` and I then convert to a 90 time to tell the spincore by dividing by sqrt(75) (75 W amplifier). The second value is ∫√(P(t))dt of the 90 pulse capture and the third value is ∫√(P(t))dt for the 180 pulse capture